### PR TITLE
[FEAT] Return nil if no record found upon searching for one

### PR DESCRIPTION
**Pull request by bestori**
Return nil instead of raising an ActiveRecord::RecordNotFound error

### CODE REVIEW

**Comment by bestori** [comment]
To comply with latest rails **.where(condition)** conventions.

**Comment by bestori** [comment]
@norman , can this one be merged please? :)

**Comment by parndt** [comment]
@bestori thanks for your patch.  Are you able to please fix up the whitespace changes (i.e. revert them to the indentation that we were using before) and remove the commented out lines of code?

**Comment by bestori** [comment]
@parndt Sure. Thank you :)

**Comment by bestori** [comment]
@parndt done. Thank you!

**Comment by bestori** [comment]
https://github.com/jruby/activerecord-jdbc-adapter/issues/700

**Comment by bestori** [comment]
@norman help please? :)

**Comment by parndt** [comment]
@norman can we please disable coveralls?

File: lib/friendly_id/finder_methods.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def exists?(conditions = :none)
     # `find`.
     # @raise ActiveRecord::RecordNotFound
     def find_by_friendly_id(id)
-      first_by_friendly_id(id) or raise ActiveRecord::RecordNotFound, "can't find record with friendly id: #{id.inspect}"
+      first_by_friendly_id(id) or nil
```
Comment (parndt) [inline lib/friendly_id/finder_methods.rb]: is this `or nil` even needed now?

**Comment by norman** [comment]
> can we please disable coveralls?

@parndt Done!

**Comment by parndt** [comment]
@norman thanks ❤️

**Comment by bestori** [comment]
@parndt can we merge...?

**Comment by parndt** [comment]
@bestori can you look at my review question? 😄

File: lib/friendly_id/finder_methods.rb (hunk 1 above)
Comment (bestori) [inline lib/friendly_id/finder_methods.rb]: Not sure.

**Pull request closed without merging**